### PR TITLE
Add GenoPad presence

### PR DESCRIPTION
**Pull request by EvilBrain**
For https://pad.genoweb.xyz

image

### CODE REVIEW

**Comment by EvilBrain** [comment]
What is the problem of Codacy Static Code Analysis ??

File: websites/G/GenoPad/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "service": "GenoPad",
+  "author": {
+    "name": "Deleted 7.12i",
+    "id": "591702135310385157"
+  },
+  "category": "other",
+  "logo": "https://i.imgur.com/QAwnyL4.png",
+  "thumbnail": "https://i.imgur.com/Tp42TcS.png",
+  "version": "0.0.4",
+  "color": "#0090ff",
```
Comment (Bas950) [inline websites/G/GenoPad/dist/metadata.json:12]: version 1.0.0

File: websites/G/GenoPad/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "GenoPad",
+  "author": {
+    "name": "Deleted 7.12i",
+    "id": "591702135310385157"
+  },
+  "category": "other",
+  "logo": "https://i.imgur.com/QAwnyL4.png",
+  "thumbnail": "https://i.imgur.com/Tp42TcS.png",
```
Comment (Bas950) [inline websites/G/GenoPad/dist/metadata.json:10]: no sure if that thumbnail is really a "Promotional banner or a screenshot of the website homepage"

File: websites/G/GenoPad/dist/presence.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+const presence = new Presence({
+    clientId: "623863060410400779"
+});
+presence.on("UpdateData", async () => {
+    const presenceData = {
+        largeImageKey: "light",
+        smallImageKey: "dark",
+        startTimestamp: new Date().getTime()
+    };
+    if (document.location.hostname === "pad.genoweb.xyz") {
```
Comment (Bas950) [inline websites/G/GenoPad/dist/presence.js:10]: useless if case

**Review by Bas950 — CHANGES_REQUESTED** [review]
dont upload js file

**Comment by ririxidev** [comment]
low quality site.

**Pull request closed without merging**